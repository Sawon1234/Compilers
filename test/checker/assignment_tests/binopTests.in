fun testTimes(a : Integer, b : Integer) : Integer
{
	return a*b;
}

fun testDivide(a : Integer, b : Integer) : Integer
{
	return a/b;
}

fun testModulo(a : Integer, b : Integer): Integer
{
	return a%b;
}

fun testAdd(a : Integer, b : Integer) : Integer
{
	return a+b;
}

fun testMinus(a : Integer, b : Integer) : Integer
{
	return a-b;
}
return 0;