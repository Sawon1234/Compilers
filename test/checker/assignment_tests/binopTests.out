fun testTimes < > ( a : Integer < > , b : Integer < > ) : Integer < > return a . times < > ( b ) ; fun testDivide < > ( a : Integer < > , b : Integer < > ) : Integer < > return a . divide < > ( b ) ; fun testModulo < > ( a : Integer < > , b : Integer < > ) : Integer < > return a . modulo < > ( b ) ; fun testAdd < > ( a : Integer < > , b : Integer < > ) : Integer < > return a . plus < > ( b ) ; fun testMinus < > ( a : Integer < > , b : Integer < > ) : Integer < > return a . minus < > ( b ) ; return 0 ;